### PR TITLE
Add cgmanifest file for Component Governance

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR will add cgmanifest.json file to be used in CI's Component Governance task.

## Related PRs or issues

[AB#77893](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77893)

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,1634 @@
...
+                 "commitHash": "68ef3d10a6e3f578d9f6a13ee65e0326b320f23c"
+             }
+         }
+     },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Cake",
+            "Version": "0.33.0"
```
Comment (MatkovIvan) [inline cgmanifest.json:26]: It must be detected automatically. Add only not-detected ones

File: cgmanifest.json [hunk 2]
```diff
@@ -0,0 +1,1634 @@
...
+                 "commitHash": "af4f0eecef83a0c6c00b26f9e12d215d0df21351"
+             }
+         }
+     },
+     {
+         "component": {
+             "type": "git",
+             "git": {
+                 "repositoryUrl": "https://github.com/microsoft/appcenter-sdk-apple.git",
+                 "commitHash": "68ef3d10a6e3f578d9f6a13ee65e0326b320f23c"
```
Comment (guperrot) [inline cgmanifest.json:17]: Are we going to update that every time?

File: cgmanifest.json [hunk 3]
```diff
@@ -0,0 +1,1634 @@
...
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.v7.AppCompat",
+            "Version": "27.0.2.1"
```
Comment (guperrot) [inline cgmanifest.json:152]: Any way to fix auto detection?
That will be most likely stale / out of sync pretty quick if it's manual.

File: cgmanifest.json (hunk 2 above)
Comment (Jamminroot) [reply cgmanifest.json:17]: That is not automated. That is to be changed on release. Probably worth mentioning it in wiki somewhere

File: cgmanifest.json (hunk 3 above)
Comment (Jamminroot) [reply cgmanifest.json]: That should be auto-detected by CG with the way it is implemented on CI now. Resolving

File: cgmanifest.json (hunk 2 above)
Comment (guperrot) [reply cgmanifest.json:17]: I don't really understand since there is a commit reference here.

Comment (Jamminroot) [reply cgmanifest.json:17]: Yeah, it is supprosed to be updated manually upon bumping version of native SDKs used. I am not sure if that is something that can be reliably automated.

Comment (MatkovIvan) [reply cgmanifest.json:17]: Automation can be done separately (and for Unity as well). @Jamminroot please make sure that manually updating in the release instruction for now. Should not be a blocker to merge this PR

**Pull request merged**